fs/archive/tarfs/__init__.py

```diff
@@ -11,6 +11,7 @@
 import datetime
 
 import six
+from fs.path import normpath
```

Comment:
Please use relative imports here (`from ...path import normpath`) for consistency.

Reply:
Whoops, meant to change that back. Let me fix now.